media-gfx/phototonic/phototonic-2.1.recipe

```diff
@@ -0,0 +1,100 @@
+SUMMARY="An image viewer and organizer built with Qt and Exiv2"
+DESCRIPTION="Features
+* Support for common image formats and GIF animation
+* Supports tagging images, and filtering images by tags (IPTC)
+* Browse thumbnails recursively down a folder tree
+* Dynamic thumbnails loading
+* Image transformation and color manipulation
+* Display image information and metadata
+* Does not depend on any desktop environment."
+HOMEPAGE="https://github.com/oferkv/phototonic"
+COPYRIGHT="2015-2018 Over Kashayov
+	2015-2018 Christopher roy Bratusek
+	2015-2018 Krzysztof Pyrkosz
+	2015-2018 Roman Chistokhodov
+	2015-2018 Thomas Lübking
+	2015-2018 Tung Le
+	2015-2018 Peter Mattern"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/oferkv/phototonic/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a8b2dbc81750efabb65a4732b1704641afd4a4f772ef9c90817871ca2a52f5ef"
+SOURCE_FILENAME="phototonic-$portVersion.tar.gz"
+PATCHES="phototonic-$portVersion.patchset"
+ADDITIONAL_FILES="phototonic.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+USER_SETTINGS_FILES="
+	settings/PhotoTonic directory
+	"
+PROVIDES="
+	phototonic$secondaryArchSuffix = $portVersion
+	app:PhotoTonic = $portVersion
+	"
+	
```

Comment:
Hmm, the empty line should be before `PROVIDES`, not before `REQUIRES`.

We should have:
```

USER_SETTINGS_FILES="
	settings/PhotoTonic directory
	"

PROVIDES="
	phototonic$secondaryArchSuffix = $portVersion
	app:PhotoTonic = $portVersion
	"
REQUIRES="
	haiku$secondaryArchSuffix
```

Thanks again, and sorry for being a little bit picky :)

Reply:
i think all is set.. please review again @fbrosson